src/plugins/auto/src/plugin_config.cpp

```diff
@@ -51,6 +51,12 @@ void PluginConfig::set_property(const ov::AnyMap& properties) {
             internal_properties[name] = val;
             // when user call set_property to set some config to plugin, we also respect this and pass through the config in this case
             user_properties[name] = val;
+            if (kv.first == ov::log::level.name()) {
```

Comment:
Any test cases cover setting log level successfully and failed?

Reply:
Added test case.